src/cljs/rems/application.cljs

```diff
@@ -602,23 +595,32 @@
                              :readonly readonly?
                              :app-id (:application/id application))]))]}]))
 
-(defn- application-licenses [application edit-application]
+(defn- application-licenses [application edit-application userid]
   (when-let [licenses (not-empty (:application/licenses application))]
-    (let [accepted-licenses (:accepted-licenses edit-application)
+    (let [application-id (:application/id application)
+          accepted-licenses (get (:application/accepted-licenses application) userid)
+          possible-commands (:application/permissions application)
           license-validations (index-by [:license-id] (:validation-errors edit-application))
```

Comment:
I thought you removed these from the backend?

Reply:
Now removed from here also.